pandas/core/series.py

```diff
@@ -2525,8 +2525,66 @@ def align(self, other, join='outer', axis=None, level=None, copy=True,
                                          limit=limit, fill_axis=fill_axis,
                                          broadcast_axis=broadcast_axis)
 
-    @Appender(generic._shared_docs['rename'] % _shared_doc_kwargs)
     def rename(self, index=None, **kwargs):
+        """Alter Series row labels or name
+
+        Function / dict values must be unique (1-to-1). Labels not contained in
+        a dict / Series will be left as-is. Extra labels listed don't throw an
+        error.
+
+        Alternatively, change ``Series.name`` with a scalar value.
+
+        See the :ref:`user guide <basics.rename>` for more.
+
+        Parameters
+        ----------
+        index : scalar, list-like, dict-like or function, optional
```

Comment:
Same here as before, I am not sure about the 'list-like'. I tried a few cases and it just errors. When looking at the code, list-likes seem to take the "changing Series.name" route, but names need to be hashable. So a tuple works, but not a list.